common/src/main/java/dev/ftb/mods/ftbquests/quest/QuestObject.java

```diff
@@ -126,7 +127,19 @@ public Collection<? extends QuestObject> getChildren() {
 	}
 
 	public boolean isCompletedRaw(TeamData data) {
-		return getChildren().stream().noneMatch(child -> !child.isOptionalForProgression() && !data.isCompleted(child));
+		FTBQuests.LOGGER.info("is completed raw: checking {}", this);
+        for (QuestObject child : getChildren()) {
+			FTBQuests.LOGGER.info("- check child {}", child);
+            if (!child.isOptionalForProgression()
+                    && !data.isExcludedByOtherQuestline(child)
+                    && !data.isCompleted(child)) {
+				FTBQuests.LOGGER.info("- not completed!");
+				return false;
+			}
+        }
+        return true;
+
+//		return getChildren().stream().noneMatch(child -> !child.isOptionalForProgression() && !data.isCompleted(child));
```

Comment:
Need yeeting

Reply:
Yeah. So do the logger info messages I added to that method :flushed: